src/ui/components/Initiatives/InitiativeCreate.js

```diff
@@ -1,118 +1,32 @@
 // @flow
 import React from "react";
 import {v4 as uuid} from "uuid";
```

Comment:
We should probably update this to use William's uuid implementation and shed this dependency: 
https://github.com/sourcecred/sourcecred/blob/master/src/util/uuid.js

Reply:
done